src/prototype_creep_startup_tasks.js

```diff
@@ -205,13 +205,12 @@ Creep.prototype.getEnergyFromStorage = function() {
 };
 
 Creep.prototype.repairStructure = function() {
-  var structure = null;
-  var range = null;
-  var i = null;
-  var structures = null;
+  let structure = null;
+  let i = null;
+  let structures = null;
 
   if (this.memory.target) {
-    var to_repair = Game.getObjectById(this.memory.target);
+    let to_repair = Game.getObjectById(this.memory.target);
```

Comment:
the `_` notation comes from my python background. Would move to camelCase so that we can activate that check at some point.

Not necessary now, but good to know.